### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cxwx**
add vim-plug

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,3 +44,14 @@ require("lazy").setup({
   },
 })
 ```
+
+### Using [vim-plug](https://github.com/folke/lazy.nvim)
```
Comment (McPatate) [inline README.md:48]:
```suggestion
### Using [vim-plug](https://github.com/junegunn/vim-plug)
```

File: README.md [hunk 2]
```diff
@@ -44,3 +44,14 @@ require("lazy").setup({
...
+```vim
+Plug 'https://github.com/huggingface/hfcc.nvim'
```
Comment (McPatate) [inline README.md:50]:
```suggestion
Plug 'huggingface/hfcc.nvim'
```

File: README.md [hunk 3]
```diff
@@ -44,3 +44,14 @@ require("lazy").setup({
...
+```
+```lua
+require('hfcc').setup({
+        api_token = "<insert your api token here>",
+        model = "bigcode/starcoder" -- can be a model ID or an http endpoint
+      })
```
Comment (McPatate) [inline README.md:56]:
```suggestion
require('hfcc').setup({
  api_token = "<insert your api token here>",
  model = "bigcode/starcoder" -- can be a model ID or an http endpoint
})
```

**Review by McPatate — CHANGES_REQUESTED** [review]
Small nits, thanks for contributing !

Does this fix #7 ?

File: README.md (hunk 3 above)
Comment (McPatate) [reply README.md:56]: Removing the `api_token` option as this is not the recommended way to authenticate anymore, cf #11

**Comment by cxwx** [comment]
> Small nits, thanks for contributing !
> 
> Does this fix #7 ?

yes

**Pull request merged**